carbon-design-system/carbon
# Allow for HTML elements in rendered <fieldset>

Closes #7797 

#### Changelog

**New**

Allows for HTML elements to be provided to the TileGroup component without forcing them into becoming `<RadioTile>` components

#### Testing / Reviewing

1. Create a TileGroup
2. Pass "Tile" type children to the TileGroup as well as any HTML element
3. Notice the tiles render as Carbon tiles, and the HTML element renders as an HTML element
